www/core/test/extension_test.xml

```diff
@@ -5,10 +5,10 @@
 		<description>Joomla! 3.10 CMS</description>
 		<element>joomla</element>
 		<type>file</type>
-		<version>3.10.10</version>
-		<infourl title="Joomla 3.10.10 Release">https://github.com/joomla/joomla-cms/releases/tag/3.10.10</infourl>
+		<version>3.10.11</version>
+		<infourl title="Joomla 3.10.11 Release">https://github.com/joomla/joomla-cms/releases/tag/3.10.11</infourl>
```

Comment:
This should be pointing to the final release URL on joomla.org

Reply:
If the old one is for 3.10.10, should it not be the same for 3.10.11?

Reply:
```suggestion
		<infourl title="Joomla 3.10.11 Release">https://www.joomla.org/announcements/release-news/5865-joomla-4-2-release.html</infourl>
```